### ISSUE TITLE
[Bug]Server can't startup

### ISSUE THREAD

**Issue by DancingSnow0517** [tracer4b/nomi-ceu#53]
Note: if you need general tech support for things like server configuration, Discord is a better venue. Please open an issue only if there is a clear bug with the pack or if you have been asked to by one of the Discord staff.

**Pack Version:**
1.3.1

**Description:**
Server Can't startyp

**Steps To Reproduce:**
download serverpack from github release and run launch.bat

**Crash Log:**
```
---- Minecraft Crash Report ----
// Lolis deobfuscated this stacktrace using MCP's stable-39 mappings.
// Hey, that tickles! Hehehe!

Time: 2022-06-19 21:24:02 GMT+08:00
Description: Exception in server tick loop

net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from dan's Crafting Tweaks (craftingtweaks)
Caused by: java.lang.NoClassDefFoundError: net/minecraft/client/entity/EntityPlayerSP
    at net.blay09.mods.craftingtweaks.addons.CraftingTweaksAddons.postInit(CraftingTweaksAddons.java:26)
    at net.blay09.mods.craftingtweaks.CraftingTweaks.postInit(CraftingTweaks.java:219)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:637)
    at sun.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
    at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
    at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
    at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
    at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
    at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
    at com.google.common.eventbus.EventBus.post(EventBus.java:217)
    at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:219)
    at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:197)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
    at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
    at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
    at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
    at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
    at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
    at com.google.common.eventbus.EventBus.post(EventBus.java:217)
    at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:136)
    at net.minecraftforge.fml.common.Loader.initializeMods(Loader.java:754)
    at net.minecraftforge.fml.server.FMLServerHandler.finishServerLoading(FMLServerHandler.java:108)
    at net.minecraftforge.fml.common.FMLCommonHandler.onServerStarted(FMLCommonHandler.java:338)
    at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:219)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:486)
    at java.lang.Thread.run(Thread.java:750)
Caused by: java.lang.ClassNotFoundException: net.minecraft.client.entity.EntityPlayerSP
    at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:419)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:352)
    ... 37 more
Caused by: net.minecraftforge.fml.common.asm.ASMTransformerWrapper$TransformerException: Exception in class transformer net.minecraftforge.fml.common.asm.transformers.SideTransformer@682bd3c4 from coremod FMLCorePlugin
    at net.minecraftforge.fml.common.asm.ASMTransformerWrapper$TransformerWrapper.transform(ASMTransformerWrapper.java:260)
    at net.minecraft.launchwrapper.LaunchClassLoader.runTransformers(LaunchClassLoader.java:279)
    at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:176)
    ... 39 more
Caused by: java.lang.RuntimeException: Attempted to load class bud for invalid side SERVER
    at net.minecraftforge.fml.common.asm.transformers.SideTransformer.transform(SideTransformer.java:62)
    at net.minecraftforge.fml.common.asm.ASMTransformerWrapper$TransformerWrapper.transform(ASMTransformerWrapper.java:256)
    ... 41 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
  Minecraft Version: 1.12.2
  Operating System: Windows 10 (amd64) version 10.0
  Java Version: 1.8.0_332, Azul Systems, Inc.
  Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Azul Systems, Inc.
  Memory: 1532848392 bytes (1461 MB) / 2091384832 bytes (1994 MB) up to 2091384832 bytes (1994 MB)
  JVM Flags: 2 total; -Xms2048M -Xmx2048M
  IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
  FML: MCP 9.42 LoliASM 5.2 Powered by Forge 14.23.5.2860 137 mods loaded, 137 mods active
       States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
       
       | State | ID                                | Version                            | Source                                                     | Signature                                |
       |:----- |:--------------------------------- |:---------------------------------- |:---------------------------------------------------------- |:---------------------------------------- |
       | LCHIJ | minecraft                         | 1.12.2                             | minecraft.jar                                              | None                                     |
       | LCHIJ | mcp                               | 9.42                               | minecraft.jar                                              | None                                     |
       | LCHIJ | FML                               | 8.0.99.99                          | forge-1.12.2-14.23.5.2860.jar                              | e3c3d50c7c986df74c645c0ac54639741c90a557 |
       | LCHIJ | forge                             | 14.23.5.2860                       | forge-1.12.2-14.23.5.2860.jar                              | e3c3d50c7c986df74c645c0ac54639741c90a557 |
       | LCHIJ | advancedrocketrycore              | 1                                  | minecraft.jar                                              | None                                     |
       | LCHIJ | creativecoredummy                 | 1.0.0                              | minecraft.jar                                              | None                                     |
       | LCHIJ | jecharacters                      | 1.12.0-3.5.1                       | JustEnoughCharacters-1.12.0-3.5.1.jar                      | None                                     |
       | LCHIJ | littletilescore                   | 1.0.0                              | minecraft.jar                                              | None                                     |
       | LCHIJ | mixinbooter                       | 4.2                                | minecraft.jar                                              | None                                     |
       | LCHIJ | foamfixcore                       | 7.7.4                              | minecraft.jar                                              | None                                     |
       | LCHIJ | fastbench                         | 1.7.3                              | FastWorkbench-1.12.2-1.7.3.jar                             | None                                     |
       | LCHIJ | actuallyadditions                 | 1.12.2-r152                        | ActuallyAdditions-1.12.2-r152.jar                          | None                                     |
       | LCHIJ | baubles                           | 1.5.2                              | Baubles-1.12-1.5.2.jar                                     | None                                     |
       | LCHIJ | actuallybaubles                   | 1.1                                | ActuallyBaubles-1.12-1.1.jar                               | None                                     |
       | LCHIJ | redstoneflux                      | 2.1.1                              | RedstoneFlux-1.12-2.1.1.1-universal.jar                    | None                                     |
       | LCHIJ | cofhcore                          | 4.6.6                              | CoFHCore-1.12.2-4.6.6.1-universal.jar                      | None                                     |
       | LCHIJ | libvulpes                         | 0.4.2.-88                          | LibVulpes-1.12.2-0.4.2-88-universal.jar                    | None                                     |
       | LCHIJ | advancedrocketry                  | 1.7.0.-232                         | AdvancedRocketry-1.12.2-1.7.0-232-universal.jar            | None                                     |
       | LCHIJ | advsolars                         | 1.0                                | advsolars-1.0.jar                                          | None                                     |
       | LCHIJ | appliedenergistics2               | rv6-stable-7-extended_life-v0.52.3 | appliedenergistics2-rv6-stable-7-extended_life-v0.52.3.jar | None                                     |
       | LCHIJ | bdlib                             | 1.14.3.12                          | bdlib-1.14.3.12-mc1.12.2.jar                               | None                                     |
       | LCHIJ | ae2stuff                          | 0.7.0.4                            | ae2stuff-0.7.0.4-mc1.12.2.jar                              | None                                     |
       | LCHIJ | endercore                         | 1.12.2-0.5.76                      | EnderCore-1.12.2-0.5.76.jar                                | None                                     |
       | LCHIJ | crafttweaker                      | 4.1.20                             | CraftTweaker2-1.12-4.1.20.673.jar                          | None                                     |
       | LCHIJ | mtlib                             | 3.0.7                              | MTLib-3.0.7.jar                                            | None                                     |
       | LCHIJ | modtweaker                        | 4.0.19                             | modtweaker-4.0.20.5.jar                                    | None                                     |
       | LCHIJ | jei                               | 4.16.1.301                         | jei_1.12.2-4.16.1.301.jar                                  | None                                     |
       | LCHIJ | codechickenlib                    | 3.2.3.358                          | CodeChickenLib-1.12.2-3.2.3.358-universal.jar              | f1850c39b2516232a2108a7bd84d1cb5df93b261 |
       | LCHIJ | brandonscore                      | 2.4.20                             | BrandonsCore-1.12.2-2.4.20.162-universal.jar               | None                                     |
       | LCHIJ | cofhworld                         | 1.4.0                              | CoFHWorld-1.12.2-1.4.0.1-universal.jar                     | None                                     |
       | LCHIJ | thermalfoundation                 | 2.6.7                              | ThermalFoundation-1.12.2-2.6.7.1-universal.jar             | None                                     |
       | LCHIJ | draconicevolution                 | 2.3.28                             | Draconic-Evolution-1.12.2-2.3.28.354-universal.jar         | None                                     |
       | LCHIJ | thermalexpansion                  | 5.5.7                              | ThermalExpansion-1.12.2-5.5.7.1-universal.jar              | None                                     |
       | LCHIJ | enderio                           | 5.3.70                             | EnderIO-1.12.2-5.3.70.jar                                  | None                                     |
       | LCHIJ | p455w0rdslib                      | 2.3.161                            | p455w0rdslib-1.12.2-2.3.161.jar                            | 186bc454cd122c9c2f1aa4f95611254bcc543363 |
       | LCHIJ | ae2wtlib                          | 1.0.34                             | AE2WTLib-1.12.2-1.0.34.jar                                 | 186bc454cd122c9c2f1aa4f95611254bcc543363 |
       | LCHIJ | appleskin                         | 1.0.14                             | AppleSkin-mc1.12-1.0.14.jar                                | None                                     |
       | LCHIJ | architecturecraft                 | @VERSION@                          | architecturecraft-1.12-3.98.jar                            | None                                     |
       | LCHIJ | thedragonlib                      | 1.12.2-5.3.0                       | thedragonlib-1.12.2-5.3.0.jar                              | None                                     |
       | LCHIJ | armorplus                         | 1.12.2-11.25.1.62                  | armorplus-1.12.2-11.25.1.62.jar                            | None                                     |
       | LCHIJ | attributefix                      | 1.0.4                              | AttributeFix-1.12.2-1.0.4.jar                              | d476d1b22b218a10d845928d1665d45fce301b27 |
       | LCHIJ | autoreglib                        | 1.3-32                             | AutoRegLib-1.3-32.jar                                      | None                                     |
       | LCHIJ | avaritia                          | 3.3.0                              | Avaritia-1.12.2-3.3.0.37-universal.jar                     | None                                     |
       | LCHIJ | base                              | 3.14.0                             | base-1.12.2-3.14.0.jar                                     | None                                     |
       | LCHIJ | betterquesting                    | 4.0.0                              | BetterQuesting-4.0.0.jar                                   | None                                     |
       | LCHIJ | bookshelf                         | 2.3.590                            | Bookshelf-1.12.2-2.3.590.jar                               | d476d1b22b218a10d845928d1665d45fce301b27 |
       | LCHIJ | forgelin                          | 1.8.4                              | Forgelin-1.8.4.jar                                         | None                                     |
       | LCHIJ | bountiful                         | 2.2.2                              | Bountiful-2.2.2.jar                                        | None                                     |
       | LCHIJ | buildinggadgets                   | 2.8.4                              | BuildingGadgets-2.8.4.jar                                  | None                                     |
       | LCHIJ | chameleon                         | 1.12-4.1.3                         | Chameleon-1.12-4.1.3.jar                                   | None                                     |
       | LCHIJ | chesttransporter                  | 2.8.8                              | ChestTransporter-1.12.2-2.8.8.jar                          | None                                     |
       | LCHIJ | chisel                            | MC1.12.2-1.0.2.45                  | Chisel-MC1.12.2-1.0.2.45.jar                               | None                                     |
       | LCHIJ | clumps                            | 3.1.2                              | Clumps-3.1.2.jar                                           | None                                     |
       | LCHIJ | contenttweaker                    | 1.12.2-4.10.0                      | ContentTweaker-1.12.2-4.10.0.jar                           | None                                     |
       | LCHIJ | craftelytra                       | 1.2                                | CraftableElytra-1.1.jar                                    | None                                     |
       | LCHIJ | cns                               | 1.0                                | CraftingNetherStar 1.12.1 -1.1.jar                         | None                                     |
       | LCHIE | craftingtweaks                    | 9.0.0                              | CraftingTweaks_1.12.2-9.0.0.jar                            | None                                     |
       | LCHI  | crafttweakerjei                   | 2.0.3                              | CraftTweaker2-1.12-4.1.20.673.jar                          | None                                     |
       | LCHI  | creativecore                      | 1.10.0                             | CreativeCore_v1.10.69_mc1.12.2.jar                         | None                                     |
       | LCHI  | cucumber                          | 1.1.3                              | Cucumber-1.12.2-1.1.3.jar                                  | None                                     |
       | LCHI  | darkutils                         | 1.8.230                            | DarkUtils-1.12.2-1.8.230.jar                               | d476d1b22b218a10d845928d1665d45fce301b27 |
       | LCHI  | patchouli                         | 1.0-23.6                           | Patchouli-1.0-23.6.jar                                     | None                                     |
       | LCHI  | deepmoblearning                   | 1.12.2-2.5.4                       | deepmoblearning-1.12.2-2.5.4-universal.jar                 | None                                     |
       | LCHI  | defaultworldgenerator-port        | 1.12-2.3                           | DefaultWorldGenerator-port-1.12-2.3.jar                    | None                                     |
       | LCHI  | gregtech                          | 2.3.1-beta                         | gregtech-1.12.2-2.3.1-beta.jar                             | None                                     |
       | LCHI  | devtech                           | 2.1.2                              | devtech-2.1.2.jar                                          | None                                     |
       | LCHI  | diethopper                        | 1.1                                | diethopper-1.1.jar                                         | None                                     |
       | LCHI  | theoneprobe                       | 1.4.28                             | theoneprobe-1.12-1.4.28.jar                                | None                                     |
       | LCHI  | dimensionaledibles                | 1.12.2-1.6.4                       | DimensionalEdibles-1.12.2-1.6.4.jar                        | None                                     |
       | LCHI  | effortlessbuilding                | 1.12.2-2.16                        | effortlessbuilding-1.12.2-2.16.jar                         | None                                     |
       | LCHI  | elevatorid                        | 1.3.14                             | ElevatorMod-1.12.2-1.3.14.jar                              | None                                     |
       | LCHI  | enderiointegrationtic             | 5.3.70                             | EnderIO-1.12.2-5.3.70.jar                                  | None                                     |
       | LCHI  | enderiobase                       | 5.3.70                             | EnderIO-1.12.2-5.3.70.jar                                  | None                                     |
       | LCHI  | enderioconduits                   | 5.3.70                             | EnderIO-1.12.2-5.3.70.jar                                  | None                                     |
       | LCHI  | enderioconduitsappliedenergistics | 5.3.70                             | EnderIO-1.12.2-5.3.70.jar                                  | None                                     |
       | LCHI  | enderioconduitsopencomputers      | 5.3.70                             | EnderIO-1.12.2-5.3.70.jar                                  | None                                     |
       | LCHI  | enderioconduitsrefinedstorage     | 5.3.70                             | EnderIO-1.12.2-5.3.70.jar                                  | None                                     |
       | LCHI  | enderiointegrationforestry        | 5.3.70                             | EnderIO-1.12.2-5.3.70.jar                                  | None                                     |
       | LCHI  | enderiointegrationticlate         | 5.3.70                             | EnderIO-1.12.2-5.3.70.jar                                  | None                                     |
       | LCHI  | enderioinvpanel                   | 5.3.70                             | EnderIO-1.12.2-5.3.70.jar                                  | None                                     |
       | LCHI  | ftblib                            | 5.4.7.2                            | FTBLib-5.4.7.2.jar                                         | None                                     |
       | LCHI  | enderiomachines                   | 5.3.70                             | EnderIO-1.12.2-5.3.70.jar                                  | None                                     |
       | LCHI  | enderiopowertools                 | 5.3.70                             | EnderIO-1.12.2-5.3.70.jar                                  | None                                     |
       | LCHI  | enderioendergy                    | 5.3.70                             | EnderIO-endergy-1.12.2-5.3.70.jar                          | None                                     |
       | LCHI  | enderstorage                      | 2.4.6.137                          | EnderStorage-1.12.2-2.4.6.137-universal.jar                | f1850c39b2516232a2108a7bd84d1cb5df93b261 |
       | LCHI  | extendedcrafting                  | 1.7.0                              | ExtendedCrafting-Nomifactory-Edition-1.7.0.jar             | None                                     |
       | LCHI  | extrautils2                       | 1.0                                | extrautils2-1.12-1.9.9.jar                                 | None                                     |
       | LCHI  | fasterladderclimbing              | 0.1-146                            | FasterLadderClimbing-1.12-0.1-146.jar                      | None                                     |
       | LCHI  | fastleafdecay                     | v14                                | FastLeafDecay-v14.jar                                      | None                                     |
       | LCHI  | foamfix                           | 0.10.14-1.12.2                     | foamfix-0.10.14-1.12.2.jar                                 | None                                     |
       | LCHI  | forgivingvoid                     | 1.1.0                              | ForgivingVoid_1.12.2-1.1.0.jar                             | None                                     |
       | LCHI  | storagedrawers                    | 5.2.2                              | StorageDrawers-1.12.2-5.4.2.jar                            | None                                     |
       | LCHI  | framedcompactdrawers              | 1.2.7                              | framedcompactdrawers-1.2.7.jar                             | None                                     |
       | LCHI  | ftbbackups                        | 1.1.0.1                            | FTBBackups-1.1.0.1.jar                                     | None                                     |
       | LCHI  | ftbutilities                      | 5.4.1.131                          | FTBUtilities-5.4.1.131.jar                                 | None                                     |
       | LCHI  | globalgamerules                   | 2.2                                | GlobalGameRules-1.12.2-2.2.7.jar                           | None                                     |
       | LCHI  | gcym                              | 1.1.2                              | GregicalityMultiblocks-1.12.2-1.1.2.jar                    | None                                     |
       | LCHI  | librarianlib                      | 4.22                               | librarianlib-1.12.2-4.22.jar                               | None                                     |
       | LCHI  | hooked                            | %VERSION%                          | hooked-1.0.3.jar                                           | None                                     |
       | LCHI  | inventorytweaks                   | 1.63+release.109.220f184           | InventoryTweaks-1.63.jar                                   | 55d2cd4f5f0961410bf7b91ef6c6bf00a766dcbe |
       | LCHI  | journeymap                        | 1.12.2-5.7.1                       | journeymap-1.12.2-5.7.1.jar                                | None                                     |
       | LCHI  | jecalculation                     | 1.12.2-3.2.7                       | JustEnoughCalculation-1.12.2-3.2.7.jar                     | None                                     |
       | LCHI  | justenoughdimensions              | 1.6.0-dev.20200416.184714          | justenoughdimensions-1.12.2-1.6.0-dev.20200416.184714.jar  | 2b03e1423915a189b8094816baa18f239d576dff |
       | LCHI  | littletiles                       | 1.5.0                              | LittleTiles_v1.5.57_mc1.12.2.jar                           | None                                     |
       | LCHI  | spark                             | 1.5.2                              | spark-forge1122.jar                                        | None                                     |
       | LCHI  | loliasm                           | 5.2                                | loliasm-5.2.jar                                            | None                                     |
       | LCHI  | lostcities                        | 2.0.22                             | lostcities-1.12-2.0.22.jar                                 | None                                     |
       | LCHI  | morefurnaces                      | 1.11.0                             | MoreFurnaces-Nomifactory-Edition-1.12.2-1.11.0.jar         | None                                     |
       | LCHI  | morelibs                          | 1.2.2                              | morelibs-1.12.2-1.2.2.jar                                  | None                                     |
       | LCHI  | moreplates                        | 5.5.2                              | moreplates-1.12.2-5.5.2.jar                                | None                                     |
       | LCHI  | morpheus                          | 1.12.2-3.5.106                     | Morpheus-1.12.2-3.5.106.jar                                | None                                     |
       | LCHI  | multiblocktweaker                 | 1.12.2-stable-gtceu-1.7.1          | multiblocktweaker-1.12.2-stable-gtceu-1.7.1.jar            | None                                     |
       | LCHI  | nonetherportals                   | 1.0.0                              | nonetherPortals_v1.0.0formc1.12.2.jar                      | None                                     |
       | LCHI  | nuclearcraft                      | 2.18y                              | NuclearCraft-2.18y-1.12.2.jar                              | None                                     |
       | LCHI  | openglider                        | @VERSION@                          | OpenGlider-1.12.1-1.1.0.jar                                | None                                     |
       | LCHI  | packagedauto                      | 1.12.2-1.0.5.19                    | PackagedAuto-1.12.2-1.0.5.19.jar                           | None                                     |
       | LCHI  | packagedexcrafting                | 1.12.2-1.0.2.3                     | PackagedExCrafting-1.12.2-1.0.2.3.jar                      | None                                     |
       | LCHI  | packmode                          | 1.2.0                              | packmode-1.12.2-1.2.0.jar                                  | None                                     |
       | LCHI  | placebo                           | 1.6.0                              | Placebo-1.12.2-1.6.0.jar                                   | None                                     |
       | LCHI  | questbook                         | 3.1.1-1.12                         | questbook-3.1.1-1.12.jar                                   | None                                     |
       | LCHI  | redstonearsenal                   | 2.6.6                              | RedstoneArsenal-1.12.2-2.6.6.1-universal.jar               | None                                     |
       | LCHI  | simplyjetpacks                    | 2.2.15.69                          | SimplyJetpacks2-1.12.2-2.2.15.69.jar                       | None                                     |
       | LCHI  | snad                              | 1.12.1-1.7.09.16a                  | Snad-1.12.1-1.7.09.16a.jar                                 | None                                     |
       | LCHI  | solarflux                         | 12.4.11                            | SolarFluxReborn-1.12.2-12.4.11.jar                         | 9f5e2a811a8332a842b34f6967b7db0ac4f24856 |
       | LCHI  | supersoundmuffler                 | 1.0.2.10                           | supersoundmuffler-revived_1.12.2_1.0.2.10.jar              | None                                     |
       | LCHI  | thermalcultivation                | 0.3.6                              | ThermalCultivation-1.12.2-0.3.6.1-universal.jar            | None                                     |
       | LCHI  | thermalinnovation                 | 0.3.6                              | ThermalInnovation-1.12.2-0.3.6.1-universal.jar             | None                                     |
       | LCHI  | tombmanygraves                    | 1.12-4.2.0                         | TombManyGraves-1.12-4.2.0.jar                              | None                                     |
       | LCHI  | topaddons                         | 1.12.2-1.13.0                      | topaddons-1.12.2-1.13.0.jar                                | None                                     |
       | LCHI  | torchmaster                       | 1.8.5.0                            | torchmaster_1.12.2-1.8.5.0.jar                             | None                                     |
       | LCHI  | trashslot                         | 8.4.10                             | TrashSlot_1.12.2-8.4.10.jar                                | None                                     |
       | LCHI  | wct                               | 3.12.97                            | WirelessCraftingTerminal-1.12.2-3.12.97.jar                | 186bc454cd122c9c2f1aa4f95611254bcc543363 |
       | LCHI  | worleycaves                       | 1.5.2                              | worleycaves-1.12.2-1.5.2.jar                               | None                                     |
       | LCHI  | xtones                            | 1.2.2                              | Xtones-1.2.2.jar                                           | None                                     |
       | LCHI  | densemetals                       | 2.0.0                              | DenseMetals-1.12.2-2.0.0.30.jar                            | d476d1b22b218a10d845928d1665d45fce301b27 |
       | LCHI  | phosphor-lighting                 | 1.12.2-0.2.6                       | phosphor-1.12.2-0.2.6+build50-universal.jar                | f0387d288626cc2d937daa504e74af570c52a2f1 |
       | LCHI  | librarianliblate                  | 4.22                               | librarianlib-1.12.2-4.22.jar                               | None                                     |
  Loaded coremods (and transformers): RBLoadingPlugin (RealBench-1.12.2-1.3.3.jar)
                                        pw.prok.realbench.asm.RBTransformer
                                      LibrarianLib Plugin (librarianlib-1.12.2-4.22.jar)
                                        com.teamwizardry.librarianlib.asm.LibLibTransformer
                                      LoliASM (loliasm-5.2.jar)
                                        
                                      ForgelinPlugin (Forgelin-1.8.4.jar)
                                        
                                      GregTechLoadingPlugin (gregtech-1.12.2-2.3.1-beta.jar)
                                        gregtech.core.GregTechTransformer
                                      Do not report to Forge! (If you haven't disabled the FoamFix coremod, try disabling it in the config! Note that this bit of text will still appear.) (foamfix-0.10.14-1.12.2.jar)
                                        pl.asie.foamfix.coremod.FoamFixTransformer
                                      CTMCorePlugin (CTM-MC1.12.2-1.0.2.31.jar)
                                        team.chisel.ctm.client.asm.CTMTransformer
                                      AdvancedRocketryPlugin (AdvancedRocketry-1.12.2-1.7.0-232-universal.jar)
                                        zmaster587.advancedRocketry.asm.ClassTransformer
                                      MixinBooter (!mixinbooter-4.2.jar)
                                        
                                      Inventory Tweaks Coremod (InventoryTweaks-1.63.jar)
                                        invtweaks.forge.asm.ContainerTransformer
                                      EnderCorePlugin (EnderCore-1.12.2-0.5.76-core.jar)
                                        com.enderio.core.common.transform.EnderCoreTransformer
                                        com.enderio.core.common.transform.SimpleMixinPatcher
                                      PhosphorFMLLoadingPlugin (phosphor-1.12.2-0.2.6+build50-universal.jar)
                                        
                                      CreativePatchingLoader (CreativeCore_v1.10.69_mc1.12.2.jar)
                                        
                                      LoadingPlugin (ResourceLoader-MC1.12.1-1.5.3.jar)
                                        lumien.resourceloader.asm.ClassTransformer
                                      JechCore (JustEnoughCharacters-1.12.0-3.5.1.jar)
                                        me.towdium.jecharacters.core.JechClassTransformer
                                      LittlePatchingLoader (LittleTiles_v1.5.57_mc1.12.2.jar)
                                        com.creativemd.littletiles.LittleTilesTransformer
  AE2 Version: stable rv6-stable-7-extended_life-v0.52.3 for Forge 14.23.5.2847
  Ender IO: No known problems detected.
            Authlib is : /D:/Users/dongl/Desktop/server/minecraft_server.1.12.2.jar
            
            !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
            !!!You are looking at the diagnostics information, not at the crash.       !!!
            !!!Scroll up until you see the line with '---- Minecraft Crash Report ----'!!!
            !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
  List of loaded APIs: * actuallyadditionsapi (34) from ActuallyAdditions-1.12.2-r152.jar
                       * ae2wtlib|API (1.0.34) from AE2WTLib-1.12.2-1.0.34.jar
                       * appliedenergistics2|API (rv6) from appliedenergistics2-rv6-stable-7-extended_life-v0.52.3.jar
                       * ArmorPlusApi (2.1) from armorplus-1.12.2-11.25.1.62.jar
                       * Base|API (1.0.0) from base-1.12.2-3.14.0.jar
                       * Baubles|API (1.4.0.2) from Baubles-1.12-1.5.2.jar
                       * BetterQuesting|API (3.2) from BetterQuesting-4.0.0.jar
                       * BetterQuesting|API2 (3.1) from BetterQuesting-4.0.0.jar
                       * BetterWithModsAPI (Beta 0.6) from AppleSkin-mc1.12-1.0.14.jar
                       * chesttransporter|api (2) from ChestTransporter-1.12.2-2.8.8.jar
                       * Chisel-API (0.0.1) from Chisel-MC1.12.2-1.0.2.45.jar
                       * ChiselAPI|Carving (0.0.1) from Chisel-MC1.12.2-1.0.2.45.jar
                       * cofhapi (2.5.0) from CoFHCore-1.12.2-4.6.6.1-universal.jar
                       * CraftingTweaks|API (4.1) from CraftingTweaks_1.12.2-9.0.0.jar
                       * ctm-api (0.1.0) from CTM-MC1.12.2-1.0.2.31.jar
                       * ctm-api-events (0.1.0) from CTM-MC1.12.2-1.0.2.31.jar
                       * ctm-api-models (0.1.0) from CTM-MC1.12.2-1.0.2.31.jar
                       * ctm-api-textures (0.1.0) from CTM-MC1.12.2-1.0.2.31.jar
                       * ctm-api-utils (0.1.0) from CTM-MC1.12.2-1.0.2.31.jar
                       * DraconicEvolution|API (1.3) from Draconic-Evolution-1.12.2-2.3.28.354-universal.jar
                       * enderioapi (4.0.0) from EnderIO-1.12.2-5.3.70.jar
                       * enderioapi|addon (4.0.0) from EnderIO-1.12.2-5.3.70.jar
                       * enderioapi|capacitor (4.0.0) from EnderIO-1.12.2-5.3.70.jar
                       * enderioapi|conduits (4.0.0) from EnderIO-1.12.2-5.3.70.jar
                       * enderioapi|farm (4.0.0) from EnderIO-1.12.2-5.3.70.jar
                       * enderioapi|redstone (4.0.0) from EnderIO-1.12.2-5.3.70.jar
                       * enderioapi|teleport (4.0.0) from EnderIO-1.12.2-5.3.70.jar
                       * enderioapi|tools (4.0.0) from EnderIO-1.12.2-5.3.70.jar
                       * enderioapi|upgrades (4.0.0) from EnderIO-1.12.2-5.3.70.jar
                       * jeresources|API (0.9.2.60) from JustEnoughResources-1.12.2-0.9.2.60.jar
                       * journeymap|client-api (1.4) from journeymap-1.12.2-5.7.1.jar
                       * journeymap|client-api-display (1.4) from journeymap-1.12.2-5.7.1.jar
                       * journeymap|client-api-event (1.4) from journeymap-1.12.2-5.7.1.jar
                       * journeymap|client-api-model (1.4) from journeymap-1.12.2-5.7.1.jar
                       * journeymap|client-api-util (1.4) from journeymap-1.12.2-5.7.1.jar
                       * JustEnoughItemsAPI (4.13.0) from jei_1.12.2-4.16.1.301.jar
                       * MouseTweaks|API (1.0) from MouseTweaks-2.10-mc1.12.2.jar
                       * Open Glider|API (0.1) from OpenGlider-1.12.1-1.1.0.jar
                       * PatchouliAPI (6) from Patchouli-1.0-23.6.jar
                       * redstonefluxapi (2.1.1) from RedstoneFlux-1.12-2.1.1.1-universal.jar
                       * StorageDrawersAPI (2.1.0) from StorageDrawers-1.12.2-5.4.2.jar
                       * StorageDrawersAPI|event (2.1.0) from StorageDrawers-1.12.2-5.4.2.jar
                       * StorageDrawersAPI|registry (2.1.0) from StorageDrawers-1.12.2-5.4.2.jar
                       * StorageDrawersAPI|render (2.1.0) from StorageDrawers-1.12.2-5.4.2.jar
                       * StorageDrawersAPI|storage (2.1.0) from StorageDrawers-1.12.2-5.4.2.jar
                       * StorageDrawersAPI|storage-attribute (2.1.0) from StorageDrawers-1.12.2-5.4.2.jar
                       * theoneprobe_api (1.4.4) from theoneprobe-1.12-1.4.28.jar
                       * wct|api (1.1) from WirelessCraftingTerminal-1.12.2-3.12.97.jar
  Extended Crafting: Nomifactory Edition: You are using a fork of Extended Crafting created for the Nomifactory modpack.
                                          If the error above is a NoSuchFieldError or a NoSuchMethodError relating to
                                          com.blakebr0.extendedcrafting,
                                          then please report to https://github.com/Nomifactory/ExtendedCrafting/issues
                                          with this crash report.
                                          Otherwise, you can ignore this message.
  AE2 Integration: IC2:OFF, GTCE:ON, RC:OFF, MFR:OFF, Waila:OFF, Mekanism:OFF, OpenComputers:OFF, THE_ONE_PROBE:ON, TESLA:OFF, CRAFTTWEAKER:ON
  Suspected Mods: dan's Crafting Tweaks (craftingtweaks), Forge Mod Loader (FML), Minecraft Forge (forge)
  Profiler Position: N/A (disabled)
  Is Modded: Definitely; Server brand changed to 'fml,forge'
  Type: Dedicated Server (map_server.txt)

```

**Screenshots:**

**Comment by Twarug** [comment]
It's because of newly updated craftingtweaker tries to load clientside code on dedicated server.
I created pull request to fix it. Now we can only wait for acceptation.

**Comment by tracer4b** [comment]
The issue can be fixed by removing the Crafting Tweaks mod from server-side.
I would release to fix the issue, but the Nomi build script died on us

**Comment by tracer4b** [comment]
Resolved in 1.3.4
